src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/TaskList/TaskListTabPanel.tsx

```diff
@@ -40,10 +40,10 @@ const TaskListTabPanel = (props: TaskListTabPanelProps) => {
 
   const auth = useAuth();
   const toast = useToast();
-  const team = project.team;
+  const { teams } = project;
 
   if (isLoading || assigneeIsLoading || !auth.user) return <LoadingIndicator />;
-  if (!team)
+  if (teams.length === 0)
     return (
       <Typography sx={{ py: '25px', textAlign: 'center' }}>
```

Comment:
<img width="586" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/1bb7d1dd-e81f-4072-a7bb-b20399ad3def">
can we make a ticket to fix this, you could fix it, but I think it'll be a good easy ticket

Reply:
what are we fixing?

Reply:
The message ab a "project can only have tasks..." looks bad, we could spruce up how it looks a little bit pretty easily and it'd be a good, self-contained frontend ticket

I can make the ticket for it

Reply:
The message ab a "project can only have tasks..." looks bad, we could spruce up how it looks a little bit pretty easily and it'd be a good, self-contained frontend ticket

I can make the ticket for it